db/migrate/20241129052526_add_unique_index_to_collaborators.rb

```diff
@@ -0,0 +1,18 @@
+class AddUniqueIndexToCollaborators < ActiveRecord::Migration[7.0]
```

Comment:
thats why I have mention multiple time always think all the possible scenarios before adding a migration this we could have added in the CreateCollaborators migration